doc/Ptz.xml

```diff
@@ -932,6 +932,70 @@ Add GeoMove</revremark>
           </varlistentry>
         </variablelist>
       </section>
+      <section>
+        <title>MoveAndStartTracking</title>
+        <para>A device can support the MoveAndStartTracking operation. The pourpose of this action is to send an an atomic command to the device: move the camera to a wanted position and then delegate the PTZ control to the tracking algorithm. With the atomic command the latency is minimized. The move position is optional and can be set in three different modes:</para>
+		<para>By GeoLocation coordinates</para>
+		<para>By a preset token</para>
+		<para>By a PTZVector position</para>
+		<para>An existing Speed argument overrides DefaultSpeed of the corresponding PTZ configuration during movement to the requested position. If spaces are referenced within the Speed argument, they shall be speed spaces supported by the PTZ node.</para> 
+		<para>If the detection and the tracking are done in the same device, an ObjectID reference can be passed as an argument, in order to specify which object should be tracked.</para>
```

Comment:
Is it a requirement for a device to support this?

Then add something like:
A device shall support this passing an ObjectID.

Another question; how would the client know if the detection and tracking is done in the same device?